#! /bin/sh /usr/share/dpatch/dpatch-run
## db_properties.dpatch.dpatch by  <jdeolive@packaging-u1040-32.dev.opengeo.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Update to analytics database config

@DPATCH@

diff -rc orig/db.properties patched/db.properties
*** orig/db.properties	2011-03-30 22:34:20.000000000 +0000
--- patched/db.properties	2011-03-30 22:35:56.000000000 +0000
***************
*** 8,15 ****
  # sample configuration for postgres
  #
  driver=org.postgresql.Driver
! url=jdbc:postgresql://localhost:54321/geoserver
! username=postgres
  password=
  defaultAutoCommit=false
  
--- 8,15 ----
  # sample configuration for postgres
  #
  driver=org.postgresql.Driver
! url=jdbc:postgresql://localhost:5432/geoserver
! username=opengeo
  password=
  defaultAutoCommit=false
  
